src/v2/ion/uiSchemaTransformer.js

```diff
@@ -178,12 +185,18 @@ const createUISchema = (transformedResp, remediationForm) => {
       'label-top': true,
       type: 'text',
     };
+
     if (ionFormField.secret === true) {
       uiSchema.type = 'password';
       uiSchema.params = {
         showPasswordToggle: true,
       };
     }
+
+    if (Array.isArray(ionFormField.options)) {
+      uiSchema.type = 'radio';
```

Comment:
curious what is the `type` value that the backend sends in this case?

Reply:
The API design doesn't specify `type`. I bet it's `string` given how we send back the `value`

Reply:
I see. Guess it would be ideal if  the API returned a type here though ? Since we are picking just one option with a radio a string might be ok.